crypto/zkp/schemes/cl/org_credential_issuer.go

```diff
@@ -153,20 +148,17 @@ func (i *OrgCredentialIssuer) verifyCommitmentsOfAttrs(commitmentsOfAttrsProofs
 	return true
 }
 
-func (i *OrgCredentialIssuer) VerifyCredentialRequest(nymProof *dlogproofs.SchnorrProof, U *big.Int,
-	UProof *qrspecialrsaproofs.RepresentationProof,
-	commitmentsOfAttrsProofs []*commitmentzkp.DFOpeningProof) bool {
-	i.U = U
+func (i *OrgCredentialIssuer) VerifyCredentialRequest(credentialRequest *CredentialRequest) bool {
+	i.U = credentialRequest.U
```

Comment:
Please use shorter parameter names, for instance replace `credentialRequest` with `req`, or even better `cr`, because it's the way to Go (:smile: ). Otherwise it reads too much like Java :)